utilities/RedcapParser.js

```diff
@@ -154,6 +167,13 @@ function REDCapConverter(redcap_json) {
     return val;
   };
 
+  removeSpecialCharacters = function(val) {
+    if (val) {
+      return val.replace(/[^a-zA-Z0-9]/g, '');
```

Comment:
Should the code be alphanumeric or should it just be numeric?

Reply:
It can be both I think. There were some existing questionnaires that have words as the code.